### ISSUE TITLE
RPI 3 Not booting

### ISSUE THREAD

**Issue by Cryosim** [debian-pi/raspbian-ua-netinst#506]
Rainbow logo. and nothing.

**Comment by Mausy5043** [comment]
Welcome to GitHub. And welcome to the `raspbian-ua-netinst`.

It would help us help you if you could provide a tad more information.

Please read the [README.md](https://github.com/debian-pi/raspbian-ua-netinst/blob/v1.1.x/README.md) and the [CONTRIBUTING.md](https://github.com/debian-pi/raspbian-ua-netinst/blob/v1.1.x/CONTRIBUTING.md)

**Comment by CesarPhlebotomo** [comment]
I am also having this issue. I followed your instructions for Linux installation, using release v1.0.9 and xzcat to explode the archive file onto the SD card. I did not customize the installer. I put the SD card in the Pi 3, plugged in an HDMI monitor, ethernet cable, mouse, keyboard, then power supply. I hoped it would 'just work', in spite of your documentation not listing support for a Pi 3, but saw the Pi hanging on the rainbow screen again (image attached).
![debug-screen](https://user-images.githubusercontent.com/5619539/36157654-4cceb1a4-10a8-11e8-8c5b-7cdb60fd3b9d.jpg)
